test/helpers/runDeployment.js

```diff
@@ -43,63 +45,71 @@ module.exports = async (account, deploy, deployed, execute, getConfig, keccak256
     await deploy('standardPoolConverter', 'StandardPoolConverter', poolToken1.address, contractRegistry.address, 1000);
 
     // initialize contract registry
-    await execute(contractRegistry.registerAddress(asciiToHex('ContractRegistry'), contractRegistry.address));
-    await execute(contractRegistry.registerAddress(asciiToHex('ConverterFactory'), converterFactory.address));
-    await execute(contractRegistry.registerAddress(asciiToHex('BancorNetwork'), bancorNetwork.address));
-    await execute(contractRegistry.registerAddress(asciiToHex('NetworkSettings'), networkSettings.address));
-
-    await execute(contractRegistry.registerAddress(asciiToHex('ConversionPathFinder'), conversionPathFinder.address));
-    await execute(contractRegistry.registerAddress(asciiToHex('BancorConverterUpgrader'), converterUpgrader.address));
-    await execute(contractRegistry.registerAddress(asciiToHex('BancorConverterRegistry'), converterRegistry.address));
+    await execute(contractRegistry.registerAddress(formatBytes32String('ContractRegistry'), contractRegistry.address));
+    await execute(contractRegistry.registerAddress(formatBytes32String('ConverterFactory'), converterFactory.address));
+    await execute(contractRegistry.registerAddress(formatBytes32String('BancorNetwork'), bancorNetwork.address));
+    await execute(contractRegistry.registerAddress(formatBytes32String('NetworkSettings'), networkSettings.address));
+
+    await execute(
+        contractRegistry.registerAddress(formatBytes32String('ConversionPathFinder'), conversionPathFinder.address)
+    );
+    await execute(
+        contractRegistry.registerAddress(formatBytes32String('BancorConverterUpgrader'), converterUpgrader.address)
+    );
+    await execute(
+        contractRegistry.registerAddress(formatBytes32String('BancorConverterRegistry'), converterRegistry.address)
+    );
     await execute(
-        contractRegistry.registerAddress(asciiToHex('BancorConverterRegistryData'), converterRegistryData.address)
+        contractRegistry.registerAddress(
+            formatBytes32String('BancorConverterRegistryData'),
+            converterRegistryData.address
+        )
     );
 
-    // initialize converter factory
     await execute(converterFactory.registerTypedConverterFactory(standardPoolConverterFactory.address));
 
-    for (const reserve of getConfig().reserves) {
+    for (const reserve of config.reserves) {
```

Comment:
Yes, of course. Please note that I also update the actual `config` object whenever it's used (instead of the previous approach of always rewriting and rereading it).